### ISSUE TITLE
[BR]: Fail to change log target on Arch Linux

### ISSUE THREAD

**Issue by ghost** [fail2ban/fail2ban#3231]
### Environment:

- Fail2Ban version  : Fail2Ban v0.11.2
- OS, including release name/version : `Linux [REDACTED] 5.16.10-arch1-1 #1 SMP PREEMPT Wed, 16 Feb 2022 19:35:18 +0000 x86_64 GNU/Linux`
- [x] Fail2Ban installed via OS/distribution mechanisms
- [x] You have not applied any additional foreign patches to the codebase
- [ ] Some customizations were done to the configuration (provide details below is so)

#### Steps to reproduce
Install fail2ban on Arch Linux

#### Expected behavior
Fail2ban starts normally
#### Observed behavior

[21435]: ERROR   NOK: ('Failed to change log target',)#### Any additional information

#### Any customizations done to /etc/fail2ban/ configuration

```
```

#### Relevant parts of /var/log/fail2ban.log file:

```
[21435]: ERROR   NOK: ('Failed to change log target',)

``` 

#### Relevant lines from monitored log files:

```
```

**Comment by sebres** [comment]
Error "Failed to change log target" is throwing in 2 reasons at the moment:
- in case your target is `SYSLOG` and something is wrong with syslog socket file
- in case your target is some file (mostly `/var/log/fail2ban.log`) and there is either no such directory or even permissions issues

In both cases you'd see another error firstly, just due to invalid logtarget you don't know probably where it is logged... Isn't something about that in /var/log/fail2ban.log?

1. How do you start fail2ban? If it is systemd-, sysv- or upstart-unit can you see which command line parameters are configured there?
Fail2ban can obtain it from fail2ban.conf (or fail2ban.local) or from command-line by parameter `--logtarget`.
2. What do you see by `fail2ban-client -d | grep logtarget`?

Basically it is a matter of Arch maintainers... 
@fail2ban/maintainers any idea what may be wrong with Arch distribution of fail2ban?

**Comment by D4v1dH03** [comment]
1. I used the default arch service configuration file and never modified since installation

2. `/var/log/fail2ban.log`
